test/spec/features/popup-menu/PopupMenuComponentSpec.js

```diff
@@ -510,6 +510,45 @@ describe('features/popup-menu - <PopupMenu>', function() {
     }));
 
 
+    it('should search description', async function() {
+
+      // given
+      createPopupMenu({ container, entries, search: true });
+
+      var searchInput = domQuery('.djs-popup-search input', container);
+      searchInput.value = 'description';
+
+      // when
+      searchInput.dispatchEvent(keyDown('ArrowUp'));
+      searchInput.dispatchEvent(keyUp('ArrowUp'));
+
+      await whenStable();
+
+      // then
+      expect(domQueryAll('.entry', container)).to.have.length(1);
+      expect(domQuery('.entry .djs-popup-label', container).textContent).to.eql('Entry 1');
+    });
+
+
+    it('should not search id', async function() {
+
+      // given
+      createPopupMenu({ container, entries, search: true });
+
+      var searchInput = domQuery('.djs-popup-search input', container);
+      searchInput.value = '6';
```

Comment:
Kinda cryptic, I stubmled over this. I'd rather call `6` `JUST_AN_ID` and set the value to that.